pom.xml

```diff
@@ -40,6 +47,13 @@
                     <linkXRef>false</linkXRef>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
+                <configuration>
```

Comment:
I'm not sure if it is necessary, but you can put version here, between 52 and 53
<version>3.2.3</version>

Reply:
Done